src/components/TransactionDetails/transactionTransformer.ts

```
@@ -0,0 +1,216 @@
+            uiStatus = 'completed'
+            break
+        case 'FAILED':
+        case 'ERROR':
+            uiStatus = 'failed'
+            break
+        case 'CANCELLED':
+        case 'EXPIRED':
+            uiStatus = 'cancelled'
+            break
+        default:
+            const knownStatuses: StatusType[] = ['completed', 'pending', 'failed', 'cancelled', 'soon', 'processing']
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)